README.md

```diff
@@ -1,7 +1,42 @@
 # Babel Plugin: Annotate React
 ​
-This repo will hold a Babel plugin that annotates React components with two attributes: `data-component` and `data-element`. These attribues will allow you to use selectors to more accurately target React-rendered components when testing and when emitting Fullstory recording events.
-​
-This plugin will work with both web and native React applications.  
-​
-* Status: This is a work-in-progress and is not yet ready for production use. *
+This repo will hold a Babel plugin that annotates React components with stable attributes that can be used to search and select using FullStory. This is most useful when using a React system that generates dynamic names for Components.
+
+For React on the web the attributes are `data-component` and `data-element`.
+
+Example input:
+
+	class HelloComponent extends Component {
+		render() {
+			return <div>
+				<h1>Hello world</h1>
+			</div>;
+		}
+	}
+
+Example output:
+
+	class HelloComponent extends Component {
```

Comment:
Would it be a good idea to outline what qualifies for a `data-component` attribute vs. `data-element`? I looked at some of the jest output and saw at least one example of a nested component (`View`) being a `data-element` value.

Reply:
`data-component` is always a React.Component name and `data-element` is either a DOM element name like `h1` or the React Native equivalent element name like `View` and `Image`. It's a bit confusing that React Native uses initial-upper names for elements but as far as I can tell the plugin output is consistent.

Reply:
Perfectly clear! Would you mind adding that bit of info to the Readme.